src/cmd/version/apiversion.go

```diff
@@ -0,0 +1,71 @@
+package version
+
+import (
+	"fmt"
+	"github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi"
+	"github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi/cloudapi"
+	"github.com/otterize/otterize-cli/src/pkg/config"
+	"github.com/otterize/otterize-cli/src/pkg/utils/prints"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+)
+
+const (
+	outputPackageKey       = "package"
+	outputFilenameKey      = "output"
+	outputFilenameKeyShort = "o"
+)
+
+var ApiVersionCmd = &cobra.Command{
+	Use:          "api-version",
+	Short:        "Get the Otterize api version",
+	Args:         cobra.NoArgs,
+	SilenceUsage: true,
+	RunE: func(_ *cobra.Command, args []string) error {
+		cloudApiVersion, err := restapi.GetAPIVersion(viper.GetString(config.OtterizeAPIAddressKey))
+		if err != nil {
+			return err
+		}
+
+		compiledApiVersion := cloudapi.Version
+
+		if viper.IsSet(outputFilenameKey) && viper.IsSet(outputPackageKey) {
+			if err := writeVersionFile(viper.GetString(outputFilenameKey), viper.GetString(outputPackageKey), cloudApiVersion); err != nil {
+				return err
+			}
+		} else {
+			prints.PrintCliOutput("Current Cloud API Version: %s", cloudApiVersion)
+			prints.PrintCliOutput("CLI was compiled with Cloud API Version: %s", compiledApiVersion)
+		}
+		return nil
+	},
+}
+
+func writeVersionFile(filename string, packageName string, version string) error {
```

Comment:
Wait I didn't realize this writes sources - I don't think this belongs in the CLI itself, as it's a build tool.

Reply:
it's a utilization of the `version` cmd. I think it's helpful. The alternative is having the cli output the version as json and add another utility to parse it and write source code, but I thought this is good enough... don't you?